apps/content/tests/test_reciters_list_api.py

```
@@ -0,0 +1,117 @@
+        self.assertNotIn("Inactive Reciter", reciter_names)
+
+        # There should be exactly one reciter (the active one with at least one valid asset)
+        self.assertEqual(1, len(items))
+
+        reciter_item = items[0]
+        self.assertEqual(self.active_reciter.id, reciter_item["id"])
+        # Only the valid RECITATION asset with READY + RECITATION resource should be counted
+        self.assertEqual(1, reciter_item["recitations_count"])
+
+    def test_list_reciters_ordering_by_name(self):
+        # Arrange – another active reciter so we can test ordering
```

Ruff (0.14.7)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)